staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -239,22 +394,33 @@ func TestUniformFlows(t *testing.T) {
 	}
 	qs := qsc.Complete(fq.DispatchingConfig{ConcurrencyLimit: 4})
 
-	exerciseQueueSetUniformScenario(t, qCfg.Name, qs, []uniformClient{
-		{1001001001, 5, 10, time.Second, time.Second},
-		{2002002002, 5, 10, time.Second, time.Second},
-	}, time.Second*20, true, true, true, true, "", clk, counter)
+	uniformScenario{name: qCfg.Name,
+		qs: qs,
+		clients: []uniformClient{
+			{1001001001, 8, 20, time.Second, time.Second - 1, false},
+			{2002002002, 8, 20, time.Second, time.Second - 1, false},
+		},
+		concurrencyLimit:     4,
+		evalDuration:         time.Second * 50,
+		expectFair:           []bool{true},
+		expectAllRequests:    true,
+		evalInqueueMetrics:   true,
+		evalExecutingMetrics: true,
+		clk:                  clk,
+		counter:              counter,
+	}.exercise(t)
 }
 
-func TestDifferentFlows(t *testing.T) {
+func TestUniformFlows3(t *testing.T) {
```

Comment:
Can you come up with a more meaningful name than "Flows3"?

Reply:
Done.